content/api/utilities/buffer.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Cypress.Buffer
+---
+
+Cypress automatically includes a [Buffer](https://github.com/feross/buffer)
+polyfill for the browser and exposes it as `Cypress.Buffer`.
+
+Use `Cypress.Buffer` to manipulate binary data, similar to the
+[nodejs Buffer class](https://nodejs.org/api/buffer.html). Instances are
+returned by `cy.readFile()` and `cy.fixture()`, and it is used with
+`.selectFile()`.
+
+## Syntax
+
+```javascript
+Cypress.Buffer.method()
+```
+
+### Usage
+
+**<Icon name="check-circle" color="green"></Icon> Correct Usage**
+
+```javascript
+Cypress.Buffer.method()
+```
+
+**<Icon name="exclamation-triangle" color="red"></Icon> Incorrect Usage**
+
+```javascript
+cy.Buffer.method() // Errors, cannot be chained off 'cy'
+```
+
+## Examples
+
+### Image Fixture
+
+#### Maniulating binary data read from disk
+
+```javascript
+// Programmatically upload the logo
+cy.readFile('images/logo.png', null).then((file) => {
+  Cypress.Buffer.isBuffer(file) // true
+})
```

Comment:
I'm concerned that if we show people `Cypress.Buffer.isBuffer(file)` like this that they might think it's going to implicitly assert like many cypress commands do. In this case, `file` will always be a `Cypress.Buffer` instance, but I'm concerned that someone might see this example, and then expect `Cypress.Buffer.isBuffer(nonBufferThing)` to fail the test like `cy.get('#non-existent-id')` would.

```suggestion
cy.readFile('images/logo.png', null).then((file) => {
  expect(Cypress.Buffer.isBuffer(file)).to.be.true
})
```